src/components/date/date.pw.tsx

```diff
@@ -130,15 +131,14 @@ test.describe("Functionality tests", () => {
   }) => {
     await mount(<DateInputCustom />);
 
-    const dayClass = `DayPicker-Day ${DAY_PICKER_PREFIX}selected ${DAY_PICKER_PREFIX}today`;
+    const dayClass = `rdp-day rdp-today`;
     const input = getDataElementByValue(page, "input");
     await input.fill(TODAY_DATE_INPUT);
 
-    const dayPicker = page
-      .getByRole("row")
-      .locator(`div[aria-label="${TODAY}"]`);
-    await expect(dayPicker).toHaveAttribute("aria-label", TODAY);
-    await expect(dayPicker).toHaveClass(dayClass);
+    const dayPicker = page.locator(`td[data-today="true"]`).getByRole("button");
+    await expect(dayPicker).toHaveAttribute("aria-label", `Today, ${TODAY}`);
+    const inputParent = page.locator(`td[data-today="true"]`);
+    await expect(inputParent).toHaveClass(dayClass);
```

Comment:
**suggestion(non-blocking)**: I noticed the React Day Picker's docs doesn't seem to cover the set `data-*` attributes. To make our tests more robust and less reilant on undocumented attributes, what do you think about using an approach like this?:

```suggestion
    const todayButton = page.getByRole("button", { name: `Today, ${TODAY}`});
    const todayCell = page.getByRole("gridcell").filter({
      has: todayButton
    });
    
    await expect(todayButton).toBeVisible();
    await expect(todayCell).toHaveClass(dayClass);
```

This approach avoids directly depending on the `data-*` tags, instead locating elements in a way similar to how assitive tech would interpret the page.